jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -36,13 +44,62 @@ export class ChromeDevToolsProxyServer {
       ws.send('hello');
     });
 
+    server.listen(SERVER_PORT);
+
+    // TODO: move callback out to standalone method
+    server.on('request', function (request: http.IncomingMessage, response: http.ServerResponse) {
+      var url = request.url || '';
+      if (url == '/json') {
+        url = '/json/list';
+      }
+
+      if (request.method != "GET" || url.slice(0, 6) != '/json/') {
+        return;
+      }
+
+      var cmd = url.slice(6);
```

Comment:
Wonder if it's better to use node's URL module for the parsing of the URL.
I can imagine the switch will break if the client adds search/query params (i.e. `/json/list?something=abc`) to the end of the URL for example.

```
import { URL } from 'url';  // comes with node.js, no need to add any new packages

const url = new URL(request.url, 'http://127.0.0.1:${wssOptions.port}/');
url.path // contains  really only the path (no query params)
```

Reply:
I'd probably also use `split('/')` instead of using `slice()` with a magic number.
```
> '/json/list'.split('/')
[ '', 'json', 'list' ]
```

Reply:
@grgustaf did you see these suggestions ^^^ ? Any comment on them?

Reply:
Thanks, yeah missed these earlier - got it covered now.